Marlin/src/module/stepper_indirection.cpp

```diff
@@ -563,7 +566,7 @@
     chopconf.hend = chopper_timing.hend + 3;
     chopconf.hstrt = chopper_timing.hstrt - 1;
     #if ENABLED(SQUARE_WAVE_STEPPING)
-      chopconf.dedge = true;
+      chopconf.dedge = true; // Not in TMCStepper v0.4.1?
```

Comment:
@teemuatlut — It looks like the `dedge` field is not in the `TMC2660_n::CHOPCONF_t` structure in v0.4.1. Can you confirm?

Reply:
For 2660 it's in `DRVCTRL_0_t`, along with mres and intepol.
But I'd suggest setting mres with the library helper to avoid rewriting the conversion from user configuration to bit pattern.

It would be good to also set `st.sdoff = 0`, even if that is the default.

Reply:
> I'd suggest setting mres with the library helper…

In general…? Or do we need to set `mres` here too?

Reply:
In Marlin I recommend using only `.microsteps()` and not `.mres`.